src/main/java/com/arpnetworking/clusteraggregator/configuration/ClusterAggregatorConfiguration.java

```diff
@@ -164,6 +169,7 @@ public String toString() {
     private ClusterAggregatorConfiguration(final Builder builder) {
         _monitoringCluster = builder._monitoringCluster;
         _monitoringService = builder._monitoringService;
+        _monitoringUri = builder._monitoringUri;
```

Comment:
This is represented as host and port in MAD:

https://github.com/ArpNetworking/metrics-aggregator-daemon/blob/master/src/main/java/com/arpnetworking/metrics/mad/configuration/AggregatorConfiguration.java#L76

Same in `master-2.0`:

https://github.com/ArpNetworking/metrics-aggregator-daemon/blob/master-2.0/src/main/java/com/inscopemetrics/mad/configuration/AggregatorConfiguration.java#L72

I think there is some debate a to whether the client should expose setting the entire URI (wrt protocol and path encapsulation).

Happy to discuss, but it seems these should be the same across the two projects.